client/src/pages/Login.js

```diff
@@ -1,67 +1,210 @@
+            setOpen(true);
+          }
+        })
+        .catch((err) => console.log(err));
+    }
   };
 
   return (
-    <div style={{ height: "100vh" }}>
-      <h1>This is our Login page.</h1>
+    <Grid container component="main" className={classes.root}>
+      <CssBaseline />
 
-      <input
-        id="email"
-        type="email"
-        value={email}
-        placeholder="email"
-        onChange={(e) => setEmail(e.target.value)}
-      />
-      <input
-        id="password"
-        type="password"
-        value={password}
-        placeholder="password"
-        onChange={(e) => setPassword(e.target.value)}
-      />
-      <button onClick={handleClick}>Login</button>
-    </div>
+      <Grid item xs={12} sm={8} md={5} component={Paper} elevation={6} square>
+        <div className={classes.paper}>
+          <Logo className={classes.avatar} />
+          <Typography component="h1" variant="h3">
+            Log In
+          </Typography>
+          <form className={classes.form}>
+            <TextField
+              variant="outlined"
+              margin="normal"
+              required
+              fullWidth
+              id="email"
+              label="Email Address"
+              name="email"
+              autoComplete="email"
+              error={errEmail !== ""}
+              helperText={errEmail}
+              autoFocus
+              onChange={(e) => setEmail(e.target.value)}
+            />
+            <TextField
+              variant="outlined"
+              margin="normal"
+              required
+              fullWidth
+              name="password"
+              label="Password"
+              type="password"
+              value={password}
+              id="password"
+              autoComplete="current-password"
+              onChange={(e) => setPassword(e.target.value)}
+            />
+            <Button
+              type="submit"
+              fullWidth
+              variant="contained"
+              color="primary"
+              className={classes.submit}
+              onClick={handleClick}
+            >
+              Sign In
+            </Button>
+            <Grid container>
+              <Grid item>
+                <Link href="/signup" variant="body2">
+                  {"Don't have an account? Sign Up"}
+                </Link>
+              </Grid>
+            </Grid>
+          </form>
+        </div>
+        <div className={classes.snackbar}>
+        <Snackbar open={open} autoHideDuration={6000} onClose={handleClose}>
+          <Alert onClose={handleClose} severity="error">
+            Error ! Your email or password is incorrect !
+          </Alert>
+        </Snackbar>
+      </div>
+      </Grid>
+      <Grid item xs={false} sm={1} md={7} className={classes.image} />
+      
+    </Grid>
   );
 }
 
 const loginUser = async (user) => {
   try {
     const res = await axios.post("/api/auth", user);
+    console.log(res);
     setAuth(res.data.token);
```

Comment:
i don't think `setAuth` needs to be in a separate function since it doesn't seem to be used anywhere else, it's a 2-liner. either just set it in here, or, set it in Context